org.kde.ghostwriter.yml

```diff
@@ -0,0 +1,95 @@
+app-id: org.kde.ghostwriter
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+base: io.qt.qtwebengine.BaseApp
+base-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: ghostwriter
+rename-icon: ghostwriter
+finish-args:
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--socket=pulseaudio"
+  - "--share=ipc"
+  - "--share=network"
+  - "--device=dri"
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+modules:
+  - name: cmark
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: git
+        url: https://github.com/commonmark/cmark.git
```

Comment:
You can use auto generated tarballs https://github.com/commonmark/cmark/archive/refs/tags/0.31.0.tar.gz

Reply:
It's source code..

Reply:
Oh now I get it.